### ISSUE TITLE
Issue with k8s.io/docs/concepts/architecture/master-node-communication/

### ISSUE THREAD

**Issue by banatara** [kubernetes/website#17880]
**This is a Bug Report**

**Problem:**
I am not clearly able to differentiate the Cluster to Master. The topic states that its about Master and Node. What do you typically mean to say Cluster and Master in many places inside the doc. It should be Master and Node or Cluster and Node. There is something illogical to say Cluster and Master, since Master is part of the Cluster and we typically mean Cluster and Master interchangeably. So I would request that clarification is provided to the reader before getting into the topic.  
**Proposed Solution:**

**Page to Update:**
https://kubernetes.io/...

**Comment by sftim** [comment]
@banatar that's a valid observation.

I would reword:
“_Cluster to Master_” → “_Node to control plane_”
“_Master to Cluster_” → “_Control plane to node_”

and make sure that the page makes it clear that it's common and typical to have several nodes (not just one).

**Note**
If anyone is picking this up, also consider renaming the page and adding a redirect. If you're not sure how to do those things you can ask for help using [Slack](https://kubernetes.slack.com/messages/C1J0BPD2M/), or open a work-in-progress PR with your question in the comments.

/help
/language en
/kind cleanup
/priority important-longterm

**Comment by abdulbasitkay** [comment]
It this issue still open to contributon?
